docs/development/compiler/configuration/Manifest.md

```diff
@@ -79,9 +79,9 @@ This is a sample file:
     the following fields:
     - **name** : Author name.
     - **email** : Author email address.
+    - **maintainer**: Boolean, can be used to mark those authors who are currently
+      maintaining the code, as opposed to previous authors who are no longer involved.
   - **version**: Version of the library.
-  - **Qooxdoo-versions**: List of versions of the Qooxdoo framework this library
```

Comment:
is there anyway to encode that the library only works starting with a particular qx release ?

Reply:
`qooxdoo-versions` has been deprecated in v6 and removed in v7 in favour of the `requires.@qooxdoo/framework` key, which is a semver range that indicates the exact range of qx versions the library  is supposed to work with.